client/src/pages/messaging/index.tsx

```diff
@@ -258,6 +260,69 @@ const Chat = () => {
       timestamp
     };
   });
+
+// added a useeffect to check whather the 
+useEffect(() => {
+  const validateLink = async () => {
+    try {
+      // FOR PRODUCTION MODE URL SHOULD BE https://chat-e2ee-2.azurewebsites.net/api/chat-link/status/${channelID}
+      // FOR RUNING LOCALLY URL SHOULD BE http://localhost:3001/api/chat-link/status/${channelID}
+      const URL = `http://localhost:3001/api/chat-link/status/${channelID}`
```

Comment:
urls should not be hard-coded. Also UI layer  does not need to know anything about the server. That's why we have the service. 
You can expose another method in `service`, and use that here.